repository/l.json

```diff
@@ -1732,6 +1732,17 @@
 				}
 			]
 		},
+		{
+			"name": "LSP-copilot",
+			"details": "https://github.com/TheSecEng/LSP-copilot",
+			"labels": ["AI", "auto-complete", "github"],
+			"releases": [
+				{
+					"sublime_text": ">4126",
```

Comment:
I guess `>=` 4126 since it's the latest stable build.

Reply:
I wonder whether we want to prepend a min version now like https://github.com/sublimelsp/repository/blob/9391a520ca86a2e7ce3599a367689d4cbbf711cb/repository.json#L59-L68 so that if we bump the min version someday in the future, it won't be looks odd.

Reply:
I wonder whether we want to prepend the min ST version to the tag at this moment like https://github.com/sublimelsp/repository/blob/9391a520ca86a2e7ce3599a367689d4cbbf711cb/repository.json#L59-L68 so that if we bump the min version, it won't looks odd.